pkgs/applications/video/freetube/default.nix

```diff
@@ -36,7 +36,7 @@ stdenv.mkDerivation rec {
   '';
 
   postFixup = ''
-    makeWrapper ${electron}/bin/electron $out/bin/${pname} \
+    makeWrapper ${electron_22}/bin/electron $out/bin/${pname} \
```

Comment:
For the future, could you please add a comment about matching the electron version with upstream to prevent breakage?

Reply:
Oh, sry, I presumed that was clear from the pr message.